src/screens/Events/EventsMap.js

```diff
@@ -2,6 +2,7 @@
 import React, { Component } from "react";
 import { StyleSheet, Text, View } from "react-native";
 import { connect } from "react-redux";
+import { Translate, withLocalize, setActiveLanguage } from "react-localize-redux";
```

Comment:
It doesn't look like `Translate` is being used in `EventsMap`, so we can remove that import.